styles/pages/main-menu.scss

```diff
@@ -499,7 +499,6 @@ $drawer-width: 768px;
 
 	&__header {
 		width: 100%;
-		height: 31px;
```

Comment:
Does outright removing the height here make sense versus just increasing it? What do these changes look like?

Reply:
Thanks! Looks good to me, we'll pull this in for the next release.